deps/oob-1.0.2/jsonutil.c

```diff
@@ -16,26 +17,29 @@
 #define TMP_BUF_LEN 40
 #endif
 
-//static struct conStr connectionInfo ;
 static struct conStr connectionInfo = {{0}, {0}, {0}, {0}, {0}, {0}, {0}, {0}, {0}, {0}, {0}, {0}, {0}, {0}, {0} };
 
 static struct logDetails oobevent = {{0}, {0}, {0}, {0}, 0, 0};
 
+cJSON *dsn = NULL;
```

Comment:
What is the correct style we adopt ? I see both `type* var` and `type *var`

Reply:
I learned it as type* var but cJSON *dsn seems to be the convention within the prepareOOBevent function call. I'd be happy to change to type* var for this file.